etna/records/migrations/0001_initial.py

```diff
@@ -27,7 +27,7 @@ class Migration(migrations.Migration):
                         to="wagtailcore.page",
                     ),
                 ),
-                ("iaid", models.IntegerField()),
+                ("metadataId", models.IntegerField()),
```

Comment:
Again, this needs to be undone - and a new migration created to account for the field change